doc/sphinx/user/install/local-installation/documentation.md

```diff
@@ -0,0 +1,33 @@
+(sec:install:local-installation:documentation)=
+# Building the documentation
+
+If you intend to modify
+ASPECT for your own experiments, you may want to also
+generate documentation about the source code and the user manual.
+This step is optional and not necessary if you do not intend to
+modify the source code or contribute to the project.
+The source code documentation can be build using the command
+
+      cd doc; make
```

Comment:
Just going through this documentation to understand how to make cookbooks now. When I do `make`, I get the following message:
```
make targets:
  aspect.tag  - generates doxygen documentation into doxygen/
  all         - generates all targets
```
Maybe pick one for completeness instead of writing just `make`?